tests/unit/crypto.test.ts

```diff
@@ -1,4 +1,5 @@
 import crypto from "crypto";
+import nodeCrypto from 'node:crypto';
```

Comment:
Do we need a node prefix import? I recond that If you want to test prefix imports that should be another test :)

Reply:
no you are right, it is generic and tested as part of console. I will remove this.